clients/fides-js/src/lib/initialize.ts

```diff
@@ -49,35 +49,43 @@ import { OverlayProps } from "../components/types";
 import { updateConsentPreferences } from "./preferences";
 import { resolveConsentValue } from "./consent-value";
 import { initOverlay } from "./consent";
-import { TcfCookieConsent } from "./tcf/types";
+import { TcfOtherConsent } from "./tcf/types";
 import { FIDES_OVERRIDE_OPTIONS_VALIDATOR_MAP } from "./consent-constants";
 import { setupExtensions } from "./extensions";
 import {
   noticeHasConsentInCookie,
   transformConsentToFidesUserPreference,
 } from "./shared-consent-utils";
+import type { Fides } from "../docs";
```

Comment:
Having a dependency on the docs in this part of the code is concerning. Can this typing be moved to a more generic/shared part of the code?

Reply:
I think that's fair - I was also surprised to find that we defined the "Fides Global" interface in this file and not our big "consent-types" file that has everything else.